sources/packages/backend/libs/integrations/src/cas/cas.service.ts

```diff
@@ -82,6 +97,73 @@ export class CASService {
     return response?.data;
   }
 
+  /**
+   * Create supplier and site.
+   * @param token authentication token.
+   * @param supplierData data to be used for supplier and site creation.
+   * @returns submitted data and CAS response.
+   */
+  async createSupplierAndSite(
```

Comment:
Can this method be in `cas-supplier.service.ts`?

Reply:
I am not sure if I am following it. This is the CAS API call to `createSupplierAndSite` and it is in the CAS service responsible for making the API calls. What would be the benefit of moving it?

Reply:
I got the point now. The idea is to have a service that has all the API calls that we make to CAS. This is that service. Thanks.